From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@flanakin__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (micflan). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

davidwrede commented:
@flanakin, I just accepted a PR with some changes in supported browsers. Can you check the docs  later this afternoon to see if we still need this PR?

Thanks,
Dave

ggailey777 commented:
@flanakin this PR has been here for a while waiting for a response, and now there are conflicts. If you don't need this PR anymore please close it, otherwise please resolve the merge conflicts.

tysonn commented:
@flanakin - Closing due to unresolved merge conflicts.

ShawnJackson committed:
edit pass: load-balancer-outbound-connections

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into load-balancer-outbound-connections

ShawnJackson committed:
edit pass: load-balancer-outbound-connections

v-viksur committed:
(ForumsTeam) Hyperlink Broken

In this documentation under the section "Monitoring your storage service" "Receive Alert Notifications" hyperlink has broken. I am requesting you to review and update the hyperlink.

v-viksur committed:
(ForumsTeam) Typo

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into load-balancer-outbound-connections

ShawnJackson committed:
edit pass: load-balancer-outbound-connections

ShawnJackson committed:
edit pass: load-balancer-outbound-connections

PatAltimore committed:
Usability feedback

PatAltimore committed:
Add feedback

ktoliver committed:
edit pass: scheduler-advanced-complexity.md

ktoliver committed:
more edits

PatAltimore committed:
Add single-page app quickstart feedback

ktoliver committed:
more edits

more edits

ktoliver committed:
Update scheduler-advanced-complexity.md

PatAltimore committed:
Add feedback

PatAltimore committed:
Acrolinx fixes

PatAltimore committed:
Fix typo

vidarmsft committed:
Updated number of regions where JD can be created

cherylmc committed:
graphics

Commit:
Total rewrite.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aasnewdatasource

Commit:
Revision

Commit:
Revision

Commit:
Update analysis-services-datasource.md

kadamajay56 committed:
(ForumsTeam)Broken Link

There is a broken link to download Azure AD version 2.0 PowerShell module. I have updated it with the appropriate link. Check once and do the needful changes.

zhiweiwangmsft committed:
Datafreshness troubleshooting doc

Datafreshness troubleshooting doc

zhiweiwangmsft committed:
Minor format

Minor format

zhiweiwangmsft committed:
update filename

update filename

PatAltimore committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into patricka-fix-b2c-quickstarts-master

zhiweiwangmsft committed:
Update TOC file

update toc file pointer

zhiweiwangmsft committed:
update grammar

grammar

ShawnJackson committed:
Update load-balancer-outbound-connections.md

PatAltimore committed:
Add feedback

zhiweiwangmsft committed:
update with exisiting faq

update link to existing faq

vidarmsft committed:
Merge branch 'master' into patch-13

zhiweiwangmsft committed:
Update file name

update file name .md

PatAltimore committed:
Add feedback

zhiweiwangmsft committed:
revert the link

revert the link

zhiweiwangmsft committed:
update grammar

grammar update

ShawnJackson committed:
Update load-balancer-outbound-connections.md

jaredr80 committed:
Fix classic module paths

Updating all 3 sections where classic modules are called

jaredr80 committed:
Add Megaport to Denver

jaredr80 committed:
Add Megaport to Denver

zhiweiwangmsft committed:
Update due to blocker

update due to blockers

zhiweiwangmsft committed:
add hyphen

add hyphen

zhiweiwangmsft committed:
remove locale

remove locale

PRMerger15 committed:
Merge pull request #34054 from jaredr80/patch-4

Fix classic module paths

PRMerger12 committed:
Merge pull request #34055 from jaredr80/patch-5

Add Megaport to Denver

PRMerger16 committed:
Merge pull request #34056 from jaredr80/patch-6

Add Megaport to Denver

Commit:
quickfix03

zhiweiwangmsft committed:
update the map

update the map

spelluru committed:
updated author

spelluru committed:
updated author

PRMerger13 committed:
Merge pull request #33871 from vidarmsft/patch-13

Updated number of regions where JD can be created

PRMerger15 committed:
Merge pull request #34070 from spelluru/adfauthor0226

author change

j-martens committed:
git commit –m “fixes MicrosoftDocs/azure-docs#5033

PRMerger12 committed:
Merge pull request #34063 from MarkusVi/quickfix03

quickfix03

rayne-wiselman committed:
fixing typo

Commit:
Merge pull request #34076 from rayne-wiselman/rayne-february27

fixing typo

rayne-wiselman committed:
updating typo in article

PRMerger13 committed:
Merge pull request #34077 from rayne-wiselman/rayne-february27-2

updating typo in article

rayne-wiselman committed:
adding info for customer feedback

PRMerger14 committed:
Merge pull request #34078 from rayne-wiselman/raynemigrate-februrary27

adding info for customer feedback

Commit:
Update analysis-services-datasource.md

rayne-wiselman committed:
renaming articles and fixing articles not in TOC

rayne-wiselman committed:
Update vmware-azure-troubleshoot-failback-reprotect.md

rayne-wiselman committed:
Update .openpublishing.redirection.json

mayanknayar committed:
Updating ASR article freshness

Freshness update for ASR articles

Commit:
Merge pull request #34081 from mayanknayar/master

Updating ASR article freshness

AnoopVasudavan committed:
adding tip

rayne-wiselman committed:
Update migrate-tutorial-on-premises-azure.md

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update vmware-azure-tutorial.md

PRMerger18 committed:
Merge pull request #34084 from AnoopVasudavan/master

adding tip

rayne-wiselman committed:
Update migrate-tutorial-on-premises-azure.md

rayne-wiselman committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into rayne-february27-3

rayne-wiselman committed:
Merge branch 'rayne-february27-3' of https://github.com/rayne-wiselman/azure-docs-pr into rayne-february27-3

Commit:
freshness

Commit:
New tutorial lesson and minor changes to two other articles.

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into aasqs

Commit:
Moving data modeling articles under Data Modeling Tutorial node. This data modeling tutorial will be removed soon.

tfitzmac committed:
added log info

Commit:
Update TOC.md

PRMerger10 committed:
Merge pull request #34090 from tfitzmac/0227log

added log info

Ja-Dunn committed:
Merge pull request #33221 from ShawnJackson/load-balancer-outbound-connections

edit pass: load-balancer-outbound-connections

GitHubber17 committed:
Merge pull request #33684 from PatAltimore/patricka-fix-b2c-quickstarts-master

Add feedback to b2c quickstarts

GitHubber17 committed:
Merge pull request #33997 from Minewiskan/aasnewdatasource

Aasnewdatasource

GitHubber17 committed:
Merge pull request #34072 from j-martens/master

git commit –m “fixes MicrosoftDocs/azure-docs#5033

Commit:
Update analysis-services-create-sample-model.md

GitHubber17 committed:
Merge pull request #34088 from Blackmist/freshness

freshness

cherylmc committed:
Update vpn-gateway-howto-point-to-site-resource-manager-portal.md

itechedit committed:
Merge pull request #34089 from Minewiskan/aasqs

Aasqs

rayne-wiselman committed:
fixing text in images

Commit:
Merge pull request #34094 from cherylmc/patch-1

Update vpn-gateway-howto-point-to-site-resource-manager-portal.md

sanjeev3 committed:
Update sdk-and-samples.md

itechedit committed:
Merge pull request #34080 from rayne-wiselman/rayne-february27-3

renaming articles and fixing articles not in TOC

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

psignoret committed:
Minor update to Azure China 21Vianet name

Commit:
Merge pull request #34097 from psignoret/patch-1

Minor update to Azure China 21Vianet name

ShawnJackson committed:
Merge pull request #33617 from ktoliver/scheduler-advanced-complexity

edit pass: scheduler-advanced-complexity.md

v-alje committed:
Merge pull request #34098 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #34095 from sanjeev3/master

Update sdk-and-samples.md - Added a link.

kadamajay56 committed:
(ForumsTeam)Broken Link

I have update a broken link for downloading the PowerShell version 2.0 module with appropriate one. Kindly review and do the changes.

itechedit committed:
Merge pull request #33933 from cherylmc/cert2

graphics

snehaamicrosoft committed:
addition of eastus and removal of preview

PRMerger20 committed:
Merge pull request #34010 from kadamajay56/patch-4

(ForumsTeam)Broken Link

Lisaco88 committed:
Merge pull request #34040 from zhiweiwangmsft/patch-11

Datafreshness troubleshooting doc

Commit:
Merge pull request #34100 from snehaamicrosoft/migrate

addition of eastus and removal of preview

Commit:
Merge pull request #33202 from v-viksur/patch-20

(ForumsTeam) Typo

PRMerger18 committed:
Merge pull request #33174 from v-viksur/patch-19

(ForumsTeam) Hyperlink Broken
